### ISSUE TITLE
ejabberdctl "started" is not working

### ISSUE THREAD

**Issue by chriso153** [processone/ejabberd#3531]
I created a new Docker image from the 21.01 ecs image. I have changed the entrypoint to just be a loop that idles so that I can exec into the running container.

I start ejabberd with `ejabberdctl foreground`.

In a separate terminal execing into the running container, I then try `ejabberdctl started`. I believe this is supposed to wait until ejabberd is running and then return with a status code of 0, but all that ever happens for me is that it sits idly by for about 90 seconds at which point it exits with a status code of 1.

If I `ps -ef`, I see that ejabberd is running and I can get the status of it with `ejabberdctl status` which shows in the output that it is started.

I don't know how to debug this any further. Any assistance would be appreciated.

**Comment by weiss** [comment]
The `ejabberdctl started` call is meant to block until _background_ startup (`ejabberdctl start`) is successfully completed.  This use case doesn't really exist if ejabberd is started in the `foreground` (your use case isn't quite clear to me either).  Anyway, if 20.01 is started by systemd, it will signal readiness to systemd, so you could check the state with `systemctl is-active ejabberd`.

**Comment by badlop** [comment]
> I then try ejabberdctl started. I believe this is supposed to wait until ejabberd is running and then return with a status code of 0, 

Exactly right.

> but all that ever happens for me is that it sits idly by for about 90

Again you're right. And that is fairly easy to reproduce:

I create a container from 21.01 image, run it, and then:

```shell
$ docker exec ejabberd bin/ejabberdctl status
The node ejabberd@298df355fbae is started with status: started
ejabberd 21.1.0 is running in that node

$ docker exec ejabberd bin/ejabberdctl started
... here it sleeps for an eternity...
```

The good news: this is fixed in the new 21.04 image:

```shell
$ docker exec ejabberd bin/ejabberdctl status
The node ejabberd@3fc0cfdb3fd9 is started with status: started
ejabberd 21.4.0 is running in that node

$ docker exec ejabberd bin/ejabberdctl started
... just a few seconds

$ echo $?
0
```
